projects/schematics/src/migrations/3_0/constructor-deprecations/data/user-register.effect.migration.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  SPARTACUS_CORE,
+  NGRX_STORE,
+  AUTH_SERVICE,
+  ACTIONS,
+  USER_CONNECTOR,
+  USER_REGISTER_EFFECT,
+} from '../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const USER_REGISTRATION_EFFECT_MIGRATION: ConstructorDeprecation = {
+  // projects\core\src\user\store\effects\user-register.effect.ts
+  class: USER_REGISTER_EFFECT,
```

Comment:
Just leaving it here, but no need to change as it will just make things confusing to read as the file name and class name differs.

As this is just a variable and would like to keep it consistent to the migration variable, I believe we do not need to add USER_REGISTERS_EFFECT as the class name contains an `s`